### PR TITLE
renamed config to example-config.yaml; updated README.md

### PR DESCRIPTION
**Pull request by edtsoi430**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,19 @@ The jaeger-mongodb plugin uses the [grpc] storage architecture to interface with
 
 6. You can then play around with the HotROD application, and the traces will be stored to the local MongoDB database that we previously set up. The traces will reflect in the Jaeger UI. If you have MongoDB compass install, you can connect to it to visualize the data.
 
+## Configurable Options
+- Below is a list of options where you can configure with the mongodb storage plugin.
+- To customize any options, set the `GRPC_STORAGE_PLUGIN_CONFIGURATION_FILE` environment variable to point to a config.yaml file with options (for example, refer to `/configs/example.config.yaml` ).
```
Comment (dmichel1) [inline README.md]: `example.config.yaml` should be `example-config.yaml`

File: configs/example-config.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+mongo_url: "mongodb://10.1.15.231:27017"
```
Comment (dmichel1) [inline configs/example-config.yaml:1]: lets replace `default-config.yaml` with this `example-config.yaml` we don't need both.

**Review by dmichel1 — COMMENTED** [review]
two comments then LGTM.

File: configs/example-config.yaml (hunk 2 above)
Comment (edtsoi430) [reply configs/example-config.yaml:1]: just removed it

File: README.md (hunk 1 above)
Comment (edtsoi430) [reply README.md]: resolved

**Review by dmichel1 — APPROVED** [review]

**Pull request merged**